data/units.js

```diff
@@ -0,0 +1 @@
+module.exports = ['em', 'ex', 'cap', 'ch', 'ic', 'rem', 'lh', 'rlh', 'vw', 'vh', 'vi', 'vb', 'vmin', 'vmax', 'cm', 'mm', 'Q', 'in', 'pt', 'pc', 'px', 'deg', 'grad', 'rad', 'turn', 's', 'ms', 'Hz', 'kHz', 'dpi', 'dpcm', 'dppx', '%'];
```

Comment:
According to [the drafts page](https://drafts.csswg.org/css-values-4/#absolute-lengths), all of these units should be case-insensitive:

> Note: Values are case-insensitive and serialize as lower case, for example 1Q serializes as 1q.

I didn't check to see if the case was ignored in the code.

Reply:
Never know that.

Do we want to allow authors to write case insensitive units? Something like `[1, 'PX']`. Maybe we have to check whether those preprocessors can handle it.

Reply:
It seems that preprocessors (less/stylus) don't care about the unit. They always manipulate on the numeric part.

Reply:
Ok, let's not worry about it then.